src/Modules/LearningTasks/Tutor.LearningTasks.API/Dtos/Activities/GuidanceDto.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Tutor.LearningTasks.API.Dtos.Activities;
+
+public class GuidanceDto
+{
+    public required string DetailInfo { get; set; }
```

Comment:
`Info` is a bad word. A better name would be `Description`.

Reply:
Ok, I will change to Description.